src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -46,14 +46,15 @@ internal class WorkloadManifestUpdater : IWorkloadManifestUpdater
             IWorkloadManifestInstaller workloadManifestInstaller,
             PackageSourceLocation packageSourceLocation = null,
             Func<string, string> getEnvironmentVariable = null,
-            bool displayManifestUpdates = true)
+            bool displayManifestUpdates = true,
+            SdkFeatureBand? sdkVersion = null)
```

Comment:
Rename to `sdkFeatureBand`

```suggestion
            SdkFeatureBand? sdkFeatureBand = null)
```